database/buildSchema/43_views_functions_triggers.sql

```diff
@@ -1,3 +1,9 @@
+-- SCHEMA to hide internal functionality from postgraphile graphql schema
+-- for example views that expose tables used in row level policies to avoid compounding
+-- permission queries will be in private schema and not accessible via graphql
```

Comment:
```suggestion
-- SCHEMA to hide internal functionality from postgraphile graphql schema
-- for example, views that expose tables used in row level policies (to avoid
-- compounding permission queries) will be in private schema and not accessible
-- via graphql
```